setup.py

```diff
@@ -101,9 +101,9 @@ def get_git_head_sha(base_dir):
 
 def get_build_version(xla_git_sha):
   version = os.getenv('TORCH_XLA_VERSION', '2.2.0')
-  if _check_env_flag('VERSIONED_XLA_BUILD', default='0'):
+  if _check_env_flag('GIT_VERSIONED_XLA_BUILD', default='TRUE'):
```

Comment:
Do you know if `nightly_release` this is going to be `TRUE` or `true`?

Reply:
It doesn't matter. `_check_env_flag` will convert everything to upper case.